packages/ui/src/components/input/input.web.tsx

```diff
@@ -0,0 +1,215 @@
+import {
+  type ComponentProps,
+  LegacyRef,
+  createContext,
+  forwardRef,
+  useContext,
+  useImperativeHandle,
+  useRef,
+  useState,
+} from 'react';
+
+import { sva } from 'leather-styles/css';
+import { SystemStyleObject } from 'leather-styles/types';
+
+import { propIfDefined } from '@leather-wallet/utils';
+
+import { createStyleContext } from '../../utils/style-context.web';
+//FIXME leather-wallet/issues#64: This is a double up of a hook that already exists in the codebase.
```

Comment:
We are only using this `useOnMount` hook once in `input`. When removing the `input` from the extension I will investigate the best place to share it.